src/graphql/localState.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import type {RootState} from '../types';
+
+const defaultState: RootState = {
+  userState: {
+    __typename: 'userState',
+    userID: '1',
+    username: 'username',
+    email: 'username@kodefox.com',
+    token: 'sometoken',
+  },
+  toastState: {
```

Comment:
This is for my demo to explain the mutation that will change the state isOpen of toastState to be true and showing it up.

Reply:
awesome!